packages/pigeon/README.md

```diff
@@ -1,15 +1,15 @@
 # Pigeon
 
-**Warning: Pigeon is experimental and unsupported.  It can be removed or changed
-at any time.**
+**Warning: Pigeon 1.0 hasn't shipped yet so breaking changes may happen still.**
```

Comment:
This is a little colloquial with the "still" at the end, maybe "Pigeon is experimental and 1.0 has not yet shipped. Breaking changes may occur."  Whatever you want though.

Reply:
I can't say experimental, I'm trying to promote it somewhere between experimental and stable since it is used but I don't want to commit to no breaking changes until we get more usage.

I took your suggestion but called it "prerelease" not "experimental".